<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>249</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>249</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>761</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Todos</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>151</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>List of todos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Category browser</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>256</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>291</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Todo info</string>
      </property>
      <widget class="QTextEdit" name="todoInfoInputText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>109</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="doneCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>92</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveTodoButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>220</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Todo content</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>271</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>271</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Due To</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Categories</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>261</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>261</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Parent</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="quitPushButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>360</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
